newIDE/app/src/ExportAndShare/SigningCredentials/CreateIosSigningCredentialsDialog/CreateIosCertificateSteps.js

```diff
@@ -0,0 +1,353 @@
+        const certificateAsBase64 = await getBase64FromFile(certificateFile);
+        const {
+          certificateSerial,
+          certificateKind,
+        } = await signingCredentialApi.uploadCertificate(
+          authenticatedUser.getAuthorizationHeader,
+          userId,
+          {
+            certificateAsBase64,
+          }
+        );
+        if (certificateKind === 'unknown') {
+          const answer = await showConfirmation({
+            title: t`Unknown certificate type`,
+            message: t`This certificate type is unknown and might not work when building the app. Are you sure you want to continue?`,
+            level: 'warning',
+            confirmButtonLabel: t`Continue`,
+            dismissButtonLabel: t`Cancel`,
+          });
+          if (!answer) return;
+        }
+        await signingCredentialApi.createCertificateP12(
+          authenticatedUser.getAuthorizationHeader,
+          userId,
+          {
+            certificateSerial,
+            certificateKind,
+            certificateRequestUuid,
+          }
+        );
+
+        setWasCertificateGenerated(true);
+      } catch (err) {
+        setCertificateError(err);
+      } finally {
+        setIsCertificateLoading(false);
+      }
+    },
+    [
+      authenticatedUser.getAuthorizationHeader,
+      certificateRequestUuid,
+      showConfirmation,
+      userId,
+    ]
+  );
+
+  const onUploadMobileProvision = React.useCallback(
+    async (mobileProvisionFile: File) => {
+      if (!userId) return;
+
+      try {
+        setLastUploadedProvisioningProfileName('');
+        setIsMobileProvisionLoading(true);
+
+        const mobileProvisionAsBase64 = await getBase64FromFile(
+          mobileProvisionFile
+        );
+        const { name } = await signingCredentialApi.uploadMobileProvision(
+          authenticatedUser.getAuthorizationHeader,
+          userId,
+          {
+            mobileProvisionAsBase64,
+          }
+        );
+        setLastUploadedProvisioningProfileName(name);
+      } catch (err) {
+        setMobileProvisionError(err);
+      } finally {
+        setIsMobileProvisionLoading(false);
+      }
+    },
+    [authenticatedUser.getAuthorizationHeader, userId]
+  );
+
+  return (
+    <ColumnStackLayout noMargin>
+      <AlertMessage
+        kind="info"
+        renderLeftIcon={() => (
+          <Apple
+            style={{
+              width: 32,
+              height: 32,
+            }}
+          />
+        )}
+        renderRightButton={() => (
+          <FlatButton
+            label={'Open Apple Developer'}
+            onClick={() =>
+              Window.openExternalURL('https://developer.apple.com/account')
+            }
+          />
+        )}
+      >
+        <Trans>
+          You need a Apple Developer account to create a certificate.
+        </Trans>
+      </AlertMessage>
+      <Text size="block-title">
+        <Trans>1) Create a Certificate Signing Request and a Certificate</Trans>
```

Comment:
I would avoid it for RTL. This screen is text heavy so I felt compelled to do this, but maybe we'll redesign it later so we can avoid having to use this numbering... this will work for a first version!